src/js/modules/articles/components/summaries/RecommendedArticles.js

```diff
@@ -60,22 +75,25 @@ const styles = {
   },
 };
 
-const RecommendedArticles = ({ classes, recommendedArticles, sections }) => {
-  // NESTED IN <Col lg={3} md={3}>
+const RecommendedArticles = ({ classes, data }) => {
+  if (data.loading) {
+    return null;
+  }
+  data = humps.camelizeKeys(data);
+  const { topRankedArticles } = data;
   return (
     <div className={classes.RecommendedArticles}>
       <Link to="/recommended" className={classes.label}>
         Recommended
       </Link>
-      {recommendedArticles.map((article, index) => {
-        const section = sections[article.sectionId];
+      {topRankedArticles.map((article, index) => {
         return (
-          <div className={classes.articleItem} key={article.id}>
+          <div className={classes.articleItem} key={index}>
```

Comment:
Erm, why not use id? You should [avoid index](https://reactjs.org/docs/lists-and-keys.html)

Reply:
I need it for numbering the articles 1 through 5. Read the next few lines.

Reply:
keep index, but use id as key